src/_playground/documentation/utils/getSourceModules.js

```diff
@@ -0,0 +1,9 @@
+const getSourceModules = (sourceModulePath) => {
```

Comment:
Minor nit, but this should be `getSourceModule` (singular).  The singular module may have multiple exported components and/or functions, but it's still just one module.